app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -453,39 +479,37 @@ public void exitReview(View view) {
      * Checks if a survey fragment is active
      */
     private boolean isReviewFragmentActive() {
-        Fragment currentFragment = this.getFragmentManager ().findFragmentById(R.id.dashboard_details_container);
-        if (currentFragment instanceof ReviewFragment) {
-            return true;
-        }
-        return false;
+        return isFragmentActive(ReviewFragment.class, R.id.dashboard_details_container);
```

Comment:
@idelcano  Why do you need to have isReviewFragmentActive and isSurveyFragmentActive? We can just remove the two methods and call isFragmentActive, right?